ads/_config.js

```diff
@@ -522,6 +522,12 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'invibes': {
+    prefetch: 'https://k.r66net.com/GetAmpLink',
+    renderStartImplemented: true,
+    consentHandlingOverride: true,
```

Comment:
I couldn't find the code, just to confirm that you have customized the ad behavior with the `getConsentState()` API within the iframe. Thanks

Reply:
Same as above, the GetAmpLink endpoint is not production ready.

```
var CONSENT_POLICY_STATE = {
  UNKNOWN: 0,
  SUFFICIENT: 1,
  INSUFFICIENT: 2,
  UNKNOWN_NOT_REQUIRED: 3
};

_ampContext.getConsentState(function (consent) {
  if (consent.consentState == CONSENT_POLICY_STATE.SUFFICIENT || consent.consentState == CONSENT_POLICY_STATE.UNKNOWN_NOT_REQUIRED) {
    consentManager.updateConsent(2);
  }
});
```

We consider updateConsent with value > 0 to be consent given and will show targeted ads. The default is consent = 0 and we will not give targeted ads.

Reply:
> @raducheptanariu Thank you for the clarification. Looks great to me. One question about the `GetAmpLink` endpoint. I tested it today, and it seems it's still not ready? Could you please double check.
> 
> It's not blocking the PR, but FYI we usually cut around every Tuesday. And the change will be live in prod one week after the pre-release cut. Thanks.


> @raducheptanariu Thank you for the clarification. Looks great to me. One question about the `GetAmpLink` endpoint. I tested it today, and it seems it's still not ready? Could you please double check.
> 
> It's not blocking the PR, but FYI we usually cut around every Tuesday. And the change will be live in prod one week after the pre-release cut. Thanks.

Hello and thanks for the feedback !

The endpoint for the client js is ready. The script is minified but you can see how the functions you referenced earlier are implemented (getConsentState, renderStart, noContentAvailable).

The ad server is not production ready though. This shouldn't be an issue though, as we require publishers be configured in our system prior to integration.